### DISCUSSION TITLE
Nice work! + Suggestions

### DISCUSSION THREAD

**Discussion by Blyth12** [anthony1x6000/ROBLOX2016stylus#1] [Ideas]
Hey! I just wanted to leave this comment here, nice work on this extension, its now the main roblox theme i use, and i love it! Some questions:
1. How far are you planning to update this? Like is this going to be continiously improved?
2. Would it be possible to make the avatar editor look like the old one (reference below)
![image](https://user-images.githubusercontent.com/86171441/124826305-98537380-df6c-11eb-8bd5-b3f4dd508e07.png)
Thanks! Kepp up the great work, this is amazing!😀

**Comment by anthony1x6000** [comment]
Thanks for the feedback! I'm not sure if changing the avatar screen that drastically is possible with only CSS, but I'll give it a shot. 
Edit: To answer your other question: I'm planning to keep updating this to look as faithful to 2016 Roblox as possible for as long as possible. Maybe at some point, the website could be so different that I couldn't fix it.

**Reply by anthony1x6000** [reply]
I just looked into it, and I don't think I can change it because I'm pretty sure that it's relying on javascript to put those [green boxes](https://i.ibb.co/y5QT9Hr/image.png) above the tile. I think I could do some other changes to the avatar page to make it look a bit more like 2016, but I don't think I can do much about the [green box stuff](https://i.ibb.co/y5QT9Hr/image.png).
Edit: Same with the avatar background. Sorry, I can't really do much about it. But I'll make some small changes to the avatar page.

**Reply by Blyth12** [reply]
That's fine, thanks for the reply. It's nice to see that it's an active project, and I'll definitely keep eyes out for any updates. Thanks!
